File: MyURJC/src/app/services/SubjectService/ResponseInfo/SubjectScheduleResponse.ts
import { Schedule } from "./Schedule";

export class SubjectScheduleResponse {
  title: string;
  schedule: Schedule[];

  constructor(title: string, schedule: Schedule[]) {
    this.title = title;
    this.schedule = schedule;
  }

  getTitle(): string {
    return this.title
  }

  getSchedule(): Schedule[] {
    return this.schedule
  }
}

File: Backend/src/main/java/com/example/controller/Responses/SubjectScheduleResponse.java
package com.example.controller.Responses;

import java.util.List;

import com.example.model.Schedule;

public class SubjectScheduleResponse {
    private String title;
    private List<Schedule> schedule;

    public SubjectScheduleResponse(String title, List<Schedule> schedule) {
        this.title = title;
        this.schedule = schedule;
    }

    public String getTitle() {
        return title;
    }

    public List<Schedule> getSchedule() {
        return schedule;
    }
}
